tests/DCPS/HelloWorld/HelloWorld.mpc

```diff
@@ -0,0 +1,36 @@
+project(*idl): dcps_test_idl_only_lib {
```

Comment:
Does it need to build a shared library and two executables?  There are examples of building a single executable which can run as either publisher or subscriber and then we don't need a library.

Reply:
I think the question is:  Where are most users going to be starting?  If most users are going to have multiple processes, then the example should have multiple processes.

Reply:
It can have multiple processes, I wasn't suggesting that it should be limited to a single process.  I meant that the same executable can be used for both.

Or if you really want to build two executables, consider just including the same generated source files in both instead of having to build a library.

Reply:
Multiple executables (I misspoke).

I think HelloWorld should show two executables but it's probably okay to skip the library.